### DISCUSSION TITLE
UI not rendering properly

### DISCUSSION THREAD

**Discussion by Tsingis** [scalar/scalar#3958] [Q&A] [answered]
I'm trying to move away from using swashbuckle and swaggergen/swaggerui in .NET 9. At least `/scalar/v1` does not look usable at all. What am I missing and doing wrong?

Following instructions from https://github.com/scalar/scalar/blob/main/documentation/integrations/dotnet.md

My full code here https://github.com/Tsingis/bitcoin-web-api/tree/feat/scalar

**Comment by antlio** [comment]
hey @Tsingis, maybe @xC0dex might be able to help here ➹

**Comment by xC0dex** [comment]
Hey @Tsingis, how does the UI look like?

**Reply by Tsingis** [reply]
Not even sure how it should look like but I'm guessing it definitely should not look like it does when I run my application and navigate to /scalar/v1 which looks really broken.

I was expecting to get somewhat similar documentation experience as with swaggergen + swaggerui but guess I'm completely lost.

**Comment by Tsingis** [comment, marked as answer]
My CSP blocking stuff. My bad. :)
